### PR TITLE
Capstone1 Holistic Med Summit

### PR DESCRIPTION
**Pull request by Nyame-Wolf**
In this pull request:
- I created responsive  home and about pages, for mobile and desktop following Cindy Shin's design on behance
-  I activated the mobile menu hamburger menu to open once click and close once the x button is clicked. Once a menu item is clicked, user is redirected to the place one is interested in.
- I updated my readme to describe the capstone project as well as acknowledge the designer.
- I recored a video describing my project. The [link](https://drive.google.com/file/d/1kQpJpN4R-i-G3r7WFVW5DH8rMZtTpxlx/view?usp=sharing)

reviewer recommendations:
- fixed the footer section to match design
- Fixed alignment of  elements in about page landing section to match the original design.
- Fix logo alignment move it to the left in line with original design
- Added one more card in the program section to match original design
- Modified partner section to match design

second reviewer recommendations:

- Fixed desktop navbar so that logo is on the left.
- centered   footer content
- centered about page images 
- Fixed speaker images

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      <footer class="footer">
+        <div><span>&copy;</span> 2022 Holistic Health annual Conference</div>
+        <div>All rights reserved.</div>
+        <div class="contacts">
+          <div class="organizer">
+            <p>Maribel Pauls</p>
+            maribelpaul@gmail.com
+          </div>
+        </div>
+      </footer>
```
Comment (MahmoudBakr23) [inline about.html]: - [ ] Please make sure to make the footer on the about page match the original design, logo on the left, and text on the right
_Your design_
footer about

File: about.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            invitation to all individuals who may have an interest as well as
+            healthcare profesionals who share in our ideologies.
+          </p>
+
+          <p>
+            Please contact us via Email for further queries or clarifications.
+          </p>
+          <p>@Holisticmed2022@KenyaInternationalConvention.org</p>
+        </div>
+      </section>
```
Comment (MahmoudBakr23) [inline about.html:99]: - [ ] Please make sure to align your elements to match the original design in the main section of the about page.
_Your design_
about

File: about.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+          China
+        </p>
+        <div class="summits">
+          <img
+            class="summit"
+            src="assets/summit1.png
+          "
+            alt=""
+          />
+          <img class="summit" src="assets/summit2.png" alt="" />
```
Comment (MahmoudBakr23) [inline about.html:124]: - [ ] Kindly adjust these two pictures at the bottom of the about page to align like the original design shows.
_Your design_
red pics

File: index.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+        </li>
+      </ul>
+    </nav>
+
+    <nav class="second-nav">
+      <div class="sort">
+        <div class="open-toggle" id="open-toggle">
+          <i class="fa-solid fa-bars"></i>
+        </div>
+        <a href="index.html"><img class="logo-img" src="assets/logo-final-use.png" alt="logo"></a>
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Kindly align the logo to the left of the navbar as the original design shows
_Your design_
navbar

File: index.html [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+          <p class="p1description">
+            Listen to the various organizers and main sponsors.
+          </p>
+        </div>
+        <div class="p1">
+          <div class="p1icon">
+            <img class="icon-p" src="assets/awards.png" alt="" />
+          </div>
+          <h3 class="p1title">Awards and Achievements</h3>
+          <p class="p1description">
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Kindly add one more card in the program section as the original design shows
_Your design_
program

File: index.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+      <h2 class="partners-title">Partners</h2>
+      <hr class="hr">
+      <div class="partner-items">
+        <div>
+          <a class="partner-link" href="https://www.gatesfoundation.org/" target="_blank" rel="noopener noreferrer"><img
+              class="gates" src="assets/BillMelindaGatesFoundationp.png" alt=""></a>
+        </div>
+        <div class="partner-container">
+          <a class="partner-link" href="https://www.opensocietyfoundations.org/" target="_blank"
+            rel="noopener noreferrer"><img class="open-s" src="assets/opensocietyfoundationsfitwidth.png" alt=""></a>
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please add the dotted black and white background or something similar to the design as the original design shows
_Your design_
speakers

File: index.html [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+    <section class="partners" id="sponsor">
+      <h2 class="partners-title">Partners</h2>
+      <hr class="hr">
+      <div class="partner-items">
+        <div>
+          <a class="partner-link" href="https://www.gatesfoundation.org/" target="_blank" rel="noopener noreferrer"><img
+              class="gates" src="assets/BillMelindaGatesFoundationp.png" alt=""></a>
+        </div>
+        <div class="partner-container">
+          <a class="partner-link" href="https://www.opensocietyfoundations.org/" target="_blank"
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please work on the sizes of the icons/logos in the partner's section to match the original design
_Your design_
partners

File: index.html [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+    </section>
+    <footer id="contact">
+      <div><span>&copy;</span> 2022 Holistic Health annual Conference</div>
+      <div>All rights reserved.</div>
+      <div class="contacts">
+        <div class="organizer">
+          <p>Maribel Pauls</p>
+          maribelpaul@gmail.com
+        </div>
+      </div>
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Kindly align the elements of the footer on the home page to match the original design, the logo on the left, and the text on the right!
_Your design_
footer home
n

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Nyame-Wolf  😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Great design following so far
- Well done writing your README
- Great presentation
- Correct Linters
- Descriptive PR and commit messages
- Great semantically written HTML
- Well done using CSS media queries

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @Nyame-Wolf  🙋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

🟡  Kindly work on the changes requested by the previeous reviewer 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+    href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <header>
+      <nav class="first-nav">
+        <ul class="top-bar">
```
Comment (rukundoeric) [inline index.html:24]: - [ ] <b>REQUIRED:</b> kindly try to make sure that the orientation of your header matches with the one in provided design. 

| Original      | Yours |
  | ----------- | ----------- |
| Screen Shot 2022-06-24 at 11 06 44 | Screenshot(4) |

File: index.html [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+            Join The movement and enjoy quality life.
+          </button>
+        </div>
+        <a id="program-link" href="#">See The Whole Program</a>
+      </section>
+
+      <section class="speakers-grid">
+        <h2 class="speakers-title">Featured Speakers</h2>
+        <hr class="hr" />
+        <div class="speakers"></div>
```
Comment (rukundoeric) [inline index.html:155]: - [ ] <b>REQUIRED:</b> Also, here as mentioned in the previous review, Please add the dotted black and white background or something similar to the design as the original design shows. 

| Original      | Yours |
  | ----------- | ----------- |
| Screenshot | Screenshot(3) |

File: index.html [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+        <div class="partner-container">
+          <p class="pitem1 partner-t">World Health organization</p>
+          <p class="partner-t">Bill and Melinda gates foundation</p>
+          <p class="partner-t">Open society foundations</p>
+          <img src="assets/usaidlogo.png" alt="" />
+          <p class="pitem1 partner-t">Bloomberg Philanthropies</p>
+        </div>
+      </section>
+      <footer id="contact">
+        <div class="fot1">
```
Comment (rukundoeric) [inline index.html:169]: - [ ] <b>REQUIRED:</b> In the footer, Kindly make sure that the content is in the center as it in the given design.

| Original      | Yours |
  | ----------- | ----------- |
| Screenshot(2) | Screenshot(5) |

File: about.html [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+          Take a look at the last two summits which took place in Kyiv and Wuhan
+          China
+        </p>
+        <div class="summits">
+          <img
+            class="summit"
+            src="assets/summit1.png
+          "
+            alt=""
+          />
```
Comment (rukundoeric) [inline about.html:123]: - [ ] <b>REQUIRED:</b> Also like mentioned in the previous review, these two pics should be aligned in the center like they are in the given design. 

| Original      | Yours |
  | ----------- | ----------- |
| Screenshot(8) | Screenshot(7) |

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hello @Nyame-Wolf 👋,

_This is an additional review as a result of the second opinion request._

- [ ] As mentioned in the previous review, you did not address most of the issues raised in the first review. there are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

Keep rocking 🚀!

**Review by Munsa1 — APPROVED** [review]
Hi @Nyame-Wolf, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

Optional
image
- [ ] Kindly align the items in the image above
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**